apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx

```diff
@@ -0,0 +1,33 @@
+import RefSubLayout from '~/layouts/ref/RefSubLayout'
+
+// @ts-expect-error
+import spec from '~/../../spec/cli_v1_commands.yaml' assert { type: 'yaml' }
+import Param from '~/components/Params'
+import Options from '~/components/Options'
+
+const CliGlobalFlagsHandler = () => {
+  return (
+    <RefSubLayout.EducationRow className="not-prose">
+      <>
+        <RefSubLayout.Details>
+          <h3 className="text-lg text-scale-1200 mb-3">Flags</h3>
+          <ul className="">
+            {spec.flags.map((flag) => {
+              console.log('flag', flag)
```

Comment:
Would this spam browser console? may be use `console.debug`

Reply:
oops

Reply:
fixed